### PR TITLE
[PFBF2-0007] Register Have No Way To Hide Keyboard #51

### LINKED ISSUE #51: [PFBF2-0007] Register Have No Way To Hide Keyboard
### Priority 
- [ ] Low
- [ ] Mid
- [x] High

### Links
Name | Link
-|-
User Story | [link]()
Figma | [link]()

### Completion
Expected | Time
-|-
Expected Start | 26-08-2020
Expected End | 27-08-2020

image

### PR DESCRIPTION
**Pull request by robbyaww**
[PFBF2-0007] Register Have No Way To Hide Keyboard 
Closes #51
***
## Description
Add function to hide keyboard and set return key to hide keyboard

## Related Links
List related PRs against other branches:

List | link
------ | ------
User Story | [link]()
Figma | [link]()
Issue | [https://github.com/inxudianz/Project-Famili/issues/51]()

## Todos
- [x] UI
- [ ] Coordinator
- [ ] Network
- [ ] Tests
- [ ] Documentation

## Screenshot / Video
- Demo Video
hide keyboard

### UI Changes
UI related changes will be shown here (can be either screenshot or video format)

### Network Status
Network related changes will be shown here (screenshot of request status)

### Test Coverage
Test coverage result for designated changes will be shown here (screenshot of coverage)

## Notes
Additional notes that you have either technical debt, blocker, etc.

### CODE REVIEW

File: MainApp/MainApp/Feature/Auth/Register/RegisterViewController.swift [hunk 1]
```diff
@@ -111,8 +111,10 @@ class RegisterViewController: MasterViewController, RegisterViewProtocol {
                           tfConfirmPassword]
         for textfield in textFields {
             textfield?.addTarget(self, action: #selector(handleField), for: .editingDidEnd)
+            textfield?.addTarget(nil, action: Selector(("firstResponderAction:")), for: .editingDidEndOnExit)
```
Comment (inxudianz) [inline MainApp/MainApp/Feature/Auth/Register/RegisterViewController.swift:114]: is this still used

**Review by inxudianz — CHANGES_REQUESTED** [review]

**Comment by inxudianz** [comment]
upload video as a gif format

File: MainApp/MainApp/Feature/Auth/Register/RegisterViewController.swift (hunk 1 above)
Comment (robbyaww) [reply MainApp/MainApp/Feature/Auth/Register/RegisterViewController.swift:114]: masih, untuk hide keyboard pas klik return

**Comment by robbyaww** [comment]
> upload video as a gif format

masih gabisa will, filenya terlalu gede

**Review by inxudianz — APPROVED** [review]

**Pull request merged**